frontend/src/charts/TableChart.tsx

```diff
@@ -173,7 +175,16 @@ export function TableChart(props: TableChartProps) {
     paddingBottom: 10,
   }
 
-  const sentenceCaseName = ['hiv', 'covid', 'copd'].some((substring) =>
+  const metricSubstrings = [
+    'hiv_diagnoses',
+    'hiv_deaths',
+    'hiv_prep',
+    'hiv_prevalence',
+    'covid',
+    'copd',
+  ]
+
+  const sentenceCaseName = metricSubstrings.some((substring) =>
```

Comment:
I would prefer sending in the typed VariableConfig as a prop, and then rather then searching for a sub string, we could trigger not lowercasing the first letter if  `VARIABLE_IDS_NEEDING_UPPERCASE.includes(props.variableConfig.variableId)`

Just in general a lot cleaner to do conditional off of complete id matches versus seeking substrings since we never know what strings these values might change to in the future